microsetta_interface/implementation.py

```diff
@@ -1084,13 +1088,18 @@ def get_source(*, account_id=None, source_id=None):
 
         if template['answered']:
             per_source_taken.append(template)
-        else:
-            # NOTE 2022-08-31: Hiding the Personal Microbiome optional survey
-            # as it was never translated into Spanish.
-            # It will continue to display for users who already took it, but
-            # is unavailable to all users who have not.
-            if template['survey_template_id'] != 5:
-                per_source_not_taken.append(template)
+
+        # since we want to allow people to update surveys over time, let's
+        # show which ones have been answered, but let's also show them in
+        # the unanswered box. this allows us to test the new functionality and
+        # offer a path for users to retake the survey.
```

Comment:
Should this comment be associated with code?

Reply:
I'll remove it. It was primarily meant as a reminder for myself during development. It holds less value now that I'm more familiar with the code.